src/Orleans/Configuration/ClientConfiguration.cs

```diff
@@ -175,7 +175,11 @@ public ClientConfiguration()
             NetInterface = null;
             Port = 0;
             DNSHostName = Dns.GetHostName();
+#if NETSTANDARD
+            DeploymentId = Environment.MachineName;
+#else
```

Comment:
Maybe we should just change it to MachineName in both cases, just because the code is ugly otherwise